yt/visualization/plot_window.py

```diff
@@ -2780,3 +2912,38 @@ def plot_2d(
         aspect=aspect,
         data_source=data_source,
     )
+
+
+def _swap_axes_extents(extent):
+    """
+    swaps the x and y extent values, preservering type of extent
+
+    Parameters
+    ----------
+    extent : tuple or list
+        the current 4-element tuple or list of unyt quantities describing the
+        plot extent. extent = (xmin, xmax, ymin, ymax).
+
+    Returns
+    -------
+    tuple or list
+        the extent axes swapped, now with (ymin, ymax, xmin, xmax).
+
+    """
+    input_type = type(extent)
+    extent = [extent[2], extent[3], extent[0], extent[1]]
+    if input_type is tuple:
+        return tuple(extent)
+    return extent
+
+
+def _check_right_handed(right_handed: bool) -> bool:
+    # temporary function to check if right_handed kwarg has been set. can
+    # remove this after full depreciation.
+    if right_handed is False:
+        issue_deprecation_warning(
+            "'right_handed' has been deprecated, use 'flip_horizontal' instead.",
```

Comment:
```suggestion
            "The 'right_handed' argument is deprecated. Use 'flip_horizontal' instead.",
```

Reply:
oops, also missed this one, next batch will have it.